opennms-base-assembly/src/main/filtered/etc/collectd-configuration.xml

```diff
@@ -93,6 +131,8 @@
     </package>
 
 
+    <collector service="Cassandra21x" class-name="org.opennms.netmgt.collectd.Jsr160Collector"/>
+    <collector service="Newts" class-name="org.opennms.netmgt.collectd.Jsr160Collector"/>
```

Comment:
Isn't "DC" redundant since everything is data collection? How about just "JMX-Newts" and "JMX-Cassandra21"?